pkg/proxy/util/iptables/traffic.go

```diff
@@ -54,21 +60,39 @@ func (n *noOpLocalDetector) IfNotLocal() []string {
 	return nil // no-op; matches all traffic
 }
 
+func (n *noOpLocalDetector) IfLocalNFT() []string {
+	return nil // no-op; matches all traffic
+}
+
+func (n *noOpLocalDetector) IfNotLocalNFT() []string {
+	return nil // no-op; matches all traffic
+}
+
 type detectLocalByCIDR struct {
-	ifLocal    []string
-	ifNotLocal []string
+	ifLocal       []string
+	ifNotLocal    []string
+	ifLocalNFT    []string
+	ifNotLocalNFT []string
```

Comment:
the interesting value that is the `cidr` or the `interfaceName` is common for both, the other are specific syntaxis details of each implementation, I wanted to change this some time ago to make it more abstract, but I couldn't find anything that improved current state :/